src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -212,13 +212,13 @@ function VisEditor(
     description: 'Open Inspector for visualization',
     testId: 'openInspectorButton',
     disableButton() {
-      return !vis.hasInspector();
+      return !vis.hasInspector || !vis.hasInspector();
```

Comment:
this will be removed in the future, as all the visualize elements will have inspector when we start using pipeline